include/spirv-tools/libspirv.h

```diff
@@ -53,7 +53,7 @@ typedef enum spv_result_t {
   SPV_ERROR_INVALID_CFG = -11,
   SPV_ERROR_INVALID_LAYOUT = -12,
   SPV_ERROR_INVALID_CAPABILITY = -13,
-  SPV_ERROR_INVALID_DATA = -14,
+  SPV_ERROR_INVALID_DATA = -14, // Indicates data rules validation failure.
```

Comment:
The style guide requires two spaces before end of line comments.

https://google.github.io/styleguide/cppguide.html#Implementation_Comments

`clang-format` is a great tool for formatting these things.